Sources/KituraNet/ClientRequest.swift

```diff
@@ -380,6 +380,14 @@ public class ClientRequest {
             self.port = isHTTPS ? 443 : 80
         }
 
+        defer {
+            do {
+                try group.syncShutdownGracefully()
+            } catch {
+                Log.error("Failed to shut down the EventLoopGroup")
```

Comment:
Please include "ClientRequest" and maybe the URL for the request?

Reply:
@ianpartridge Sure! Thanks :)